content/en/serverless/guide/layer_not_authorized.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Troubleshooting Serverless Layer not Authorized Errors
+kind: documentation
+---
+This guide helps you troubleshoot the deployment error `not authorized to perform: lambda:GetLayerVersion on resource`. This error is commonly seen with Datadog Library Layers or the Datadog Extension layer.
+
+## Regionality
+Lambda functions can only include [Lambda Layers][1] which are located in the same region as the function. Typically this error occurs when users copy instrumentation settings from other applications deployed in different regions.
+
+Verify that the Lambda Layer region and Lambda function version match. Then verify that the version number is correct.
+
+You can verify that a Lambda Layer version exists using any valid AWS credentials and running the following command:
+```
+aws lambda get-layer-version \
+  --layer-name arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Node16-x \
+  --version-number {{< latest-lambda-layer-version layer="node" >}}
+
```

Comment:
```suggestion
```

let's have one (and only one) example, or this will be confusing

Reply:
That sounds good to me! Let's then explicitly say "For example, to verify the Datadog Extension layer and the Datadog Node.js library layer, run:" in line 14.

Reply:
Perfect, thank you!